syftlib/src/main/java/org/openmined/syft/execution/Plan.kt

```diff
@@ -96,10 +95,13 @@ class Plan(val job: SyftJob, val planId: String) {
      * @return the absolute path of the file containing the TorchScript model.
      */
     private fun saveScript(filesDir: String, obj: com.google.protobuf.ByteString): String {
-        val fileWriter = FileWriter(filesDir, "torchscript_${planId}.pt")
```

Comment:
We have access to job here right! Why not use joblocaldatasource to write the torchscript file?